inginious/frontend/plugins/statistics/pages/course_admin_statistics.html

```diff
@@ -77,4 +77,17 @@ <h4>Submissions Verdict (BEST)</h4>
     var adminStatistics = {
         courseId: "$:course.get_id()"
     };
+
+    // Checking if csv download function exist in the code
+    var download_csv_buttons = document.getElementsByClassName('download_csv');
+    for(let it = 0; it < download_csv_buttons.length; it++){
+        element = download_csv_buttons[it];
+        fun_name = element.getAttribute('onclick');
+
+        // windown['function']. To use check function from a string.
+        if( window[fun_name] !== "function" ) {
+            element.style.visibility = "hidden";
+        }
+    }
+    
 </script>
```

Comment:
I was testing this but the link was hidden but I had some submissions, so I guess I should be shown to me and I can download the file. So, I do not understand why we are doing this or what are the cases we should hide the option. Can you clarify this o fix it?

Reply:
This check if the function is declared. This is to solve "Download CSV option does not work, if it is not available, it shouldn't be shown to the user" issue.